expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +248,22 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
```

Comment:
> If you get it right, it will count a reversal and change the direction to up.

It shouldn't change the direction and get easier though -- it should stay at the bottom.